CHANGELOG.md

```diff
@@ -7,8 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## Unreleased
 ### Added
 - [#7 - Product now has condition attribute that is displayed on product detail page and is generated in google feed](https://github.com/shopsys/demoshop/pull/7)
+- [#8 - Category now has second description attribute that is displayed on the product list page above the product list](https://github.com/shopsys/demoshop/pull/8)
 
 ### Changed
 - [#1 - Basic changes in docs, readme etc. after copying from project-base](https://github.com/shopsys/demoshop/pull/1) : [@LukasHeinz]
 
+### Fixed
```

Comment:
There should be no "Fixed" entry to changelog, I see entry in "Added" part and it is enough, isn't it?

Reply:
I decided to add a note to the fixed section because of this commit 03a30c0d87a479db89a692f857814e300854fac3